src/index.ts

```diff
@@ -1,14 +1,18 @@
 // inspired by https://github.com/parro-it/electron-google-oauth
-import { remote, BrowserWindow } from 'electron';
+import { shell, BrowserWindow, remote } from 'electron';
 import { EventEmitter } from 'events';
 import { OAuth2Client } from 'google-auth-library';
 import { Credentials } from 'google-auth-library/build/src/auth/credentials';
 import { google } from 'googleapis';
 import { stringify } from 'querystring';
 import * as url from 'url';
+import LoopbackRedirectServer from './LoopbackRedirectServer';
 
 const BW: typeof BrowserWindow = process.type === 'renderer' ? remote.BrowserWindow : BrowserWindow;
 
+// can't be randomozed
+const LOOPBACK_INTERFACE_REDIRECTION_PORT = 42813;
```

Comment:
I know that we cannot randomize it yet, but shouldn't this hard coded value be one layer up (not inside this project)?

Reply:
Yes, indeed, I'll add some options and leave a default value.